Local.md

```diff
@@ -1,94 +1,121 @@
-
-
 # Local System Setup [macOS]
 
-Initial setup: 
-* Setup Mac with your Apple account
-* Enable FindMyMac
-* Hardware encryption with backup key stored in iCloud
-
-
-* Safari (Default Browser)
-* RayCast
-* Endpoint Management
-
-### Terminal
-* iTerm2
-* Zsh
-* Oh-My-Zsh
-* Powerline
-
-Plugins:
-* git
-* Ruby + Rails
-* autojump
-* Duplicate tab on iTerm2
-
-https://towardsdatascience.com/the-ultimate-guide-to-your-terminal-makeover-e11f9b87ac99
-
-
-### Development Environment Setup
-* Xcode
-* Ruby
-* Redis
-* Postgres App
 
+Items marked with ':heavy_dollar_sign:' are paid products or services.
 
-### Programming Tools
-* GitHub Desktop
-* Postico
-* Chrome
+Items marked with ':balance_scale:' are products or services that have both free and paid versions.
 
 
-### Code Editor
-* Atom / SublimeText /  VSCode
-* Plugins: 
-* TabNine
-* RuboCop Autocorrect
-
-### Communication
-* Slack
-* Apple Calendar
-* Fantastical (paid calendar alternative)
-* Apple Mail - Archive when done. Aim to have inbox zero. Unsubscribe from unwanted newsletters and marketing emails.
-* Superhuman (paid mail alternative)
-* Loom
-* Around
-* Zoom
-
-
-### Organisation
-* Apple Notes App
-* Bear (paid notes app alternative)
-* Apple Reminders App
-* 1Password
-* Swish
-
-### Online Tools
-* Google Docs
-* Google Slides
-* Google Sheets
-* Dropbox Paper
-* Figma
-* Miro
-* Jira
-* Linear
-* GitHub
-
+## Initial Setup ☁️
+* Setup Mac with your Apple account
+* Enable FindMyMac
+* Hardware encryption with backup key stored in iCloud
 
-### Mobile Apps to Install
-* Work Calendar 
-* Work Email
-* Authenticator app for 2FA
+|Name|Details|
+|--|--|
+|[Safari](https://www.apple.com/safari/)|Faster speed and more privacy that alternatives| 
+|[RayCast](https://raycast.com)| Powerful alternative to Spotlight|
+|[Endpoint Management](https://go.jamfnow.com/)|Enroll the computer to our company device management system.|
```

Comment:
you don't have to link this. you can add in description to ask IT for an invite to set this up. nothing that can be setup on someone on their own here. we don't want them to setup their own Jamf account by signing up on the site